backend/internal/service/handler/reviews/get_user_review_of_track.go

```diff
@@ -0,0 +1,25 @@
+package reviews
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"platnm/internal/errs"
+)
+
+func (h *Handler) GetUserReviewOfTrack(c *fiber.Ctx) error {
+
+	userId := c.Params("userId")
+	mediaId := c.Params("mediaId")
+	typeString := c.Query("media_type")
```

Comment:
honestly I think we can take this typeString out, just because the route already includes /track !